frontend/src/Components/BusinessCard/BusinessCard.css

```diff
@@ -0,0 +1,63 @@
+.business-card {
+  background-color: white;
+  border-radius: 10px;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+  overflow: hidden;
+  cursor: pointer;
```

Comment:
using cursor: pointer doesn't really make sense to me since the card isn't clickable (only a few things *within* the card are clickable)